conanfile.py

```diff
@@ -40,55 +38,16 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def _system_package_architecture(self):
-        if not tools.cross_building(self.settings):
-            return ""
-
-        if tools.os_info.with_apt:
-            if self.settings.arch == "x86":
-                return ":i386"
-            elif self.settings.arch == "x86_64":
-                return ":amd64"
-            elif self.settings.arch == "armv6" or self.settings.arch == "armv7":
-                return ":armel"
-            elif self.settings.arch == "armv7hf":
-                return ":armhf"
-            elif self.settings.arch == "armv8":
-                return ":arm64"
-
-        if tools.os_info.with_yum:
-            if self.settings.arch == "x86":
-                return ".i686"
-            elif self.settings.arch == "x86_64":
-                return ".x86_64"
-        return ""
-
-    def _system_package_name(self, package):
-        return package + self._system_package_architecture()
-
-    def system_requirements(self):
-        pack_names = []
-        if tools.os_info.with_apt:
-            pack_names.append(self._system_package_name("xkb-data"))
-            if self.options.with_x11:
-                pack_names.extend([self._system_package_name("libxcb-xkb-dev"),
-                                   self._system_package_name("libxcb1-dev")])
-        elif tools.os_info.with_yum:
-            pack_names.append(self._system_package_name("xkeyboard-config"))
-            if self.options.with_x11:
-                pack_names.append(self._system_package_name("libxcb-devel"))
-
-        if pack_names:
-            installer = tools.SystemPackageTool()
-            for item in pack_names:
-                installer.install(item)
-
     def build_requirements(self):
         if not tools.which("meson"):
-            self.build_requires("meson_installer/0.49.0@bincrafters/stable")
+            self.build_requires("meson/0.52.0")
         if not tools.which("bison"):
             self.build_requires("bison_installer/3.2.4@bincrafters/stable")
 
+    def requirements(self):
+        if self.options.with_x11:
+            self.requires("libxcb/1.13.1@bincrafters/stable")
```

Comment:
I'm confused by the list of system_requirements vs this

xkb-data/xkeyboard-config IS included in libxcb as well, but the system packages libxcb-xkb-dev/libxcb1-dev/libxcb-devel to NOT include the system packages xkb-data/xkeyboard-config?